### ISSUE TITLE
wow client freeeze and very high cpu on login

### ISSUE THREAD

**Issue by dansanti** [TrinityCore/TrinityCore#17218]
**Description**:

After create a patch for client version 21742, if you try to do login, just say connecting, and client freeze,  and cpu process increase 

**Current behaviour**: Can't login, only get connecting, and  cpu usage is igh

**Expected behaviour**: Login, and play

**Steps to reproduce the problem**:

1. create a connection pathc for client 21742 (32 or 64)
2. try to connect 

**Branch(es)**: 6.x 

**TC hash/commit**:  https://github.com/TrinityCore/TrinityCore/commit/cd51e27e195c67bc9f2e3a38c1e5e8a5a243e6bf

**TDB version**:  

**Operating system**:  Ubuntu server 14.04.4

**Comment by Aokromes** [comment]
cannot reproduce on cd51e27e195c

**Comment by dansanti** [comment]
Hi! i don0t know how debug client, but on server seems not be logs, and , if i shutdown bnetserver and try again, no client crash on login intent, but if start bnetserver, client crash

**Comment by tkrokli** [comment]
TrinityCore rev. fde9e8985f20 2016-05-30 18:31:30 +0200 (6.x branch) (Win64, Release)
TDB 6.04 + updates up to and including `2016_05_30_04_world.sql`
Operating system: Windows 10 Home edition (x64) version 1511 (OS Build 10586.318)

No problems logging in, as long as my account was correctly created according to the wiki guide -- and I also had to remember to use an adapted version of `\WTF\Config.wtf` with `SET portal "127.0.0.1"`. To be on the safe side, I also added `SET accountName "tkrokli@trinity"`. Works as intended, even on a laptop.
